keyboards/zykrah/fuyu/fuyu.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
This file should be removed as `info.json` has matrix positions

Reply:
How would the kb.c file with the LED definitions work? Since it has the include in it. Should I just move the LED devotions to the data driven model too?

Reply:
Can just include quantum, or further cut it down to whatever core headers are required.

Reply:
`#include QMK_KEYBOARD_H` would probably work too. 

Also, the leg config can be moved to the info.json file, as well.